packages/rspack-test-tools/tests/configCases/parsing/override-strict/rspack.config.js

```diff
@@ -0,0 +1,25 @@
+/** @type {import("@rspack/core").Configuration[]} */
+module.exports = [
+	{
+		mode: "production",
+		entry: ["./strict"],
+		module: {
+			parser: {
+				'javascript/auto': {
```

Comment:
This is a little different with webpack. Using `parser.javascript` does not work for this case: https://github.com/webpack/webpack/pull/18555/files#diff-7dbe21b12d5bd21e85c33e276954bcb7e6fc06371746dc76db783d77d8b4eb91R8